aspnetcore/mvc/controllers/areas/60samples/RPareas/Areas/Products/Pages/About.cshtml.cs

```diff
@@ -0,0 +1,11 @@
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.Docs.Samples;
+
+namespace RPareas.Areas.Products.Pages;
+
+public class AboutModel : PageModel
+{
+    public ContentResult OnGet() =>
+     Content(PageContext.MyDisplayRouteInfoRP().ToString()!);
```

Comment:
`MyDisplayRouteInfoRP` returns an `IActionResult`, so this should work:

```suggestion
    public IActionResult OnGet() =>
        PageContext.MyDisplayRouteInfoRP();
```